.github/workflows/deploy.yml

```diff
@@ -0,0 +1,81 @@
+name: Deploy to Azure
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+env:
+  NODE_VERSION: '18.x' # set this to the node version to use (supports 8.x, 10.x, 12.x)
+  RESOURCE_GROUP_NAME: dtmergebot3
+  FUNCTION_APP_NAME: dtmergebot3
+  STORAGE_ACCOUNT_NAME: dtmergebot3b1b7
+  STORAGE_CONTAINER_NAME: deployment
+  FUNCTION_ZIP_NAME: function.zip
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+
+      - uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - uses: pnpm/action-setup@fe02b34f77f8bc703788d5817da081398fad5dd2 # v4.0.0
+
+      - name: Install and test
+        run: |
+          pnpm install
+          pnpm run --if-present build
+          pnpm run --if-present test
+      - name: Bundle
+        working-directory: packages/mergebot
+        run: pnpm run bundle
+
+      - name: Check bundle
+        run: TYPESCRIPT_BOT_TOKEN=secret node packages/mergebot/dist/functions/index.js
+
+      - name: Create zip
+        run: zip -r ${{ env.FUNCTION_ZIP_NAME }} packagaes/mergebot/dist packages/mergebot/host.json packages/mergebot/package.json
```

Comment:
Will need to check this; the put into the zip may now be nested because we're using longer paths as the input. Better would be to consistently use `working-directory`, then output to `${{ env.GITHUB_WORKSPACE }}/${{ env.FUNCTION_ZIP_NAME }}`.